src/ansys/aedt/core/circuit.py

```diff
@@ -1412,19 +1411,19 @@ def load_diff_pairs_from_file(self, input_file):
         ----------
         >>> oDesign.LoadDiffPairsFromFile
         """
-        if not os.path.isfile(input_file):  # pragma: no cover
+        if not Path(input_file).is_file():  # pragma: no cover
             raise ValueError(f"{input_file}: The specified file could not be found.")
 
         try:
-            new_file = os.path.join(os.path.dirname(input_file), generate_unique_name("temp") + ".txt")
+            new_file = Path(input_file).parent / str(generate_unique_name("temp") + ".txt")
             with open_file(input_file, "r") as file:
                 filedata = file.read().splitlines()
             with io.open(new_file, "w", newline="\n") as fh:
                 for line in filedata:
                     fh.write(line + "\n")
 
-            self.odesign.LoadDiffPairsFromFile(new_file)
-            os.remove(new_file)
+            self.odesign.LoadDiffPairsFromFile(str(new_file))
+            Path(new_file).unlink()
```

Comment:
```suggestion
            new_file.unlink()
```
The variable is already a Path